src/main/java/Duke.java

```diff
@@ -1,50 +1,125 @@
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-        System.out.println("Hello! I'm Duke");
-        System.out.println("What can I do for you?");
-
+        showGreetings();
         String command = "";
         Scanner in = new Scanner(System.in);
 
-        int taskNum = 0;                            // number of tasks stored
-        Task[] storedTasks = new Task[100];         // array of stored texts
+        Task[] storedTasks = new Task[LIST_SIZE];         // array of stored texts
         while (!command.equals("bye")) {
+            printLineBreak();
             command = in.nextLine();
-            String[] commandWords = command.split(" ");
-            if (command.equals("list")) {
-                System.out.println("To-do: ");
-                System.out.println("________________________________________");
-                for(int i = 0; i < taskNum; i++)
-                {
-                    System.out.println((i + 1) + ". "+ storedTasks[i].getStatusIcon() + " " + storedTasks[i].getDescription());
-                }
-                System.out.println("________________________________________");
-            } else if (commandWords[0].equals("mark")) {
-                int taskIndex = Integer.parseInt(commandWords[1]) - 1;
-                storedTasks[taskIndex].setDone(true);
-                System.out.println("Nice! I've marked this task as done.");
-                System.out.println(storedTasks[taskIndex].getStatusIcon() + " " + storedTasks[taskIndex].getDescription());
-            } else if (commandWords[0].equals("unmark")) {
-                int taskIndex = Integer.parseInt(commandWords[1]) - 1;
-                storedTasks[taskIndex].setDone(false);
-                System.out.println("OK, I've marked this task as not done yet.");
-                System.out.println(storedTasks[taskIndex].getStatusIcon() + " " + storedTasks[taskIndex].getDescription());
-            } else if (!command.equals("bye")) {
-                System.out.println("added: " + command);
-                storedTasks[taskNum] = new Task(command);
-                taskNum++;
+            String[] commandLine = command.split(" ", 2);
+
+            printLineBreak();
+
+            switch (commandLine[0]) {
+            case "todo":
+                addTodoTask(storedTasks, commandLine);
+                break;
+            case "deadline":
+                addDeadlineTask(storedTasks, commandLine);
+                break;
+            case "event":
+                addEventTask(storedTasks, commandLine);
+                break;
+            case "mark":
+                markTask(storedTasks, commandLine);
+                break;
+            case "unmark":
+                unmarkTask(storedTasks, commandLine);
+                break;
+            case "list":
+                listTasks(storedTasks);
+                break;
             }
 
```

Comment:
Don't randomly leave empty line, if so must be consistent (eg. leaving an empty line between every function in the class)